src/rdkv/impl.c

```diff
@@ -20,12 +20,9 @@
 #include <ctype.h>
 #include <fcntl.h>
 #include <errno.h>
-#include <rbus/rbus.h>
```

Comment:
do not remove rbus.h
remove, rbus_value.h etc

Reply:
Changes are done